### PR TITLE
Wipe out disk partitions if we failed to install.

### PR DESCRIPTION
**Pull request by cloudnautique**
We do not try to do anything if the drive is mounted.

### CODE REVIEW

**Comment by cloudnautique** [comment]
@ibuildthecloud can you sanity check.

File: scripts/set-disk-partitions [hunk 1]
```diff
@@ -12,8 +12,14 @@ fi
 
 PARTITION_COUNT=$(grep $(echo $DEVICE | cut -d '/' -f3) /proc/partitions | wc -l)
 if [ "$PARTITION_COUNT" -gt "1" ]; then
-    echo "Device already partitioned!!! Please use a new device" 1>&2
-    exit 1
+    echo "Device already partitioned!"
+    echo "Checking to see if it is mounted"
+    if [ "$(df |grep ${DEVICE}|wc -l)" -gt "0" ]; then
```
Comment (ibuildthecloud) [inline scripts/set-disk-partitions]: Might be safer to check /proc/mounts as `df` can hang in some situations.  This statement can just become `if grep -q "${DEVICE}" /proc/mounts; then`

Comment (cloudnautique) [inline scripts/set-disk-partitions]: Yeah good call. Have definitely seen it hang with network mounts.

**Comment by cloudnautique** [comment]
@ibuildthecloud sanity check please.

**Pull request merged**